modules/nw-traditional-sharding.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ingress-cluster-traffic-ingress-controller.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-traditional-sharding_{context}"]
+== Traditional sharding example
+
+An Ingress Controller `finops-router` is configured with the label selector `spec.namespaceSelector.matchLabels.name` set to `finance` and `ops`:
+
+.Example YAML definition for `finops-router`
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name: finops-router
+  namespace: openshift-ingress-operator
+spec:
+  namespaceSelector:
+    matchLabels:
+      name:
+        - finance
+        - ops
```

Comment:
Same as the above(There was error when tried to create the ingress-controller with the doc example):
```
spec:
  namespaceSelector:
    matchExpressions:
    - key: name
      operator: In
      values:
      - finance
      - ops
```